Gradle installation note <2.0.x> [9882] (#208)
eProsima/Fast-RTPS-docs: docs/installation/sources/sources_linux.rst

    cd Fast-DDS-Gen
    gradle assemble

.. note::

>     If errors occur during compilation or you do not wish to install gradle, an executable script is included which will
>     download a gradle temporarily for the compilation step.

Reviewer: Same for MAC
```suggestion
    If errors occur during compilation, or *Gradle* installation is wished to be avoided, an executable script is included which will
    download a temporal *Gradle* for the compilation step.
```
